Book page

TeriDibbits opened the pull request:
Added default layout using Jekyll
Added back button
Added content using hierarchy
Added book room button

TeriDibbits committed:
added default layout

TeriDibbits committed:
added content

using style tile to create content
added image
added back button
added book room button

MJholland reviewed (changes requested):
add a push-1-2 to the h3 so that the h3 is closer to the p for better proximity. push-2 the whole div so it's got more space from the footer.

TeriDibbits committed:
requested changes

added push-0 to eliminate large space inbetween features and listed features

change H1 push-1-2 instead so that the space isn't so large.

TeriDibbits committed:
added push-2 to the bottom as requested

TeriDibbits commented:
Made requested spacing changes.

TeriDibbits committed:
saved changes
